app/src/main/java/ai/elimu/analytics/EventListActivity.kt

```diff
@@ -37,15 +38,29 @@ class EventListActivity : AppCompatActivity() {
         }
 
         val recyclerView = binding.recyclerview
-        val eventListAdapter = EventListAdapter(this)
+        eventListAdapter = EventListAdapter(this)
         recyclerView.adapter = eventListAdapter
         val linearLayoutManager = LinearLayoutManager(this)
         recyclerView.layoutManager = linearLayoutManager
         val dividerItemDecoration =
             DividerItemDecoration(recyclerView.context, linearLayoutManager.orientation)
         recyclerView.addItemDecoration(dividerItemDecoration)
 
-        // Fetch event counts from database, and update adapter
+        window.apply {
+            setLightStatusBar()
+            setStatusBarColorCompat(R.color.colorPrimaryDark)
+        }
+    }
+
+    override fun onResume() {
+        super.onResume()
+        fetchLearningEvents()
+    }
+
+    /**
+     * Fetch event counts from database, and update adapter
+     */
+    private fun fetchLearningEvents() {
```

Comment:
@tuancoltech Can rename since technically the list also contains _assessment_ events:

```suggestion
    private fun fetchEvents() {
```

Reply:
@jo-elimu Done.